wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -50,124 +54,162 @@ const SwipingContainer = (props) => {
                 isHost: isHost,
                 roomId: roomId,
                 name: name
-                }
-              }}/>
-            : null }
-          { noMatch ?
+              }
+            }} />
+            : null}
+          {noMatch ?
             <Redirect to={{
               pathname: '/noMatch',
               state: {
                 isHost: isHost,
                 roomId: roomId,
-                name: name}
-            }}/> : null }
+                name: name
+              }
+            }} /> : null}
           <Row>
             <Col>
-            {props.location.state.topTenMovies && (
-              <StyledDiv flex alignItems="center" marginTop={2}>
-                <CloseIcon
-                  style={{ color: theme.colors.red, fontSize: '60px'}}
-                  onClick={onClickDislike}
-                />
-                <StyledDiv padding={2}>
-                  <MovieDetail
-                    title={title}
-                    year={year}
-                    lengthOfMovie={lengthOfMovie}
-                    rating={rating}
-                    mediaType={mediaType}
-                    imageURL={imageURL}
-                    description={description}
+              {props.location.state.topTenMovies && (
+                <StyledDiv flex alignItems="center" marginTop={2}>
+                  <CloseIcon
+                    style={{ color: theme.colors.red, fontSize: '60px' }}
+                    onClick={onClickDislike}
+                  />
+                  <StyledDiv padding={2}>
+                    <MovieDetail
+                      title={title}
+                      year={year}
+                      lengthOfMovie={lengthOfMovie}
+                      rating={rating}
+                      mediaType={mediaType}
+                      imageURL={imageURL}
+                      description={description}
+                    />
+                    <StyledDiv>
+                      { isHost ?
+                      <button className={'button mt-20'} type="button" onClick={endSessionEmit}>End Session</button>
+                      : null
+                      }
+                    </StyledDiv>
+
+                  </StyledDiv>
+                  <InsertEmoticonIcon
+                    style={{ color: theme.colors.green, fontSize: '60px' }}
+                    fontSize="large"
+                    onClick={onClickLike}
                   />
                 </StyledDiv>
-                <InsertEmoticonIcon
-                  style={{ color: theme.colors.green, fontSize: '60px'}}
-                  fontSize="large"
-                  onClick={onClickLike}
-                />
-              </StyledDiv>
-            )}
+              )}
             </Col>
           </Row>
         </Container>
-    </Layout>
-  );}
+      </Layout>
+    );
+  }
   else if (SwipingCompleted) {
-    return(<SwipingCompletedScreen/>)
+    return (<SwipingCompletedScreen />)
```

Comment:
nit: can remove space here

Reply:
hmm i think all these added spaces are some sort of code style configuration that VS code has? usually we put spaces after a `return` and for self-closing components, such as this one `<Component />`

Reply:
We also usually have spaces in curly brackets like `{ matchedMovie: matchedMovie }` between elements, so I feel that these extra added spaces are enforcing code style in the code

Reply:
oh true, I just wanted to keep it consistent with what he had so far. But if thats the enforced code style then we should go with that!